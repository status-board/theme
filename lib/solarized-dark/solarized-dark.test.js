"use strict";

var _index = _interopRequireDefault(require("./index"));

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { "default": obj }; }

test('Solarized Dark should match snapshot', function () {
  expect(_index["default"]).toMatchSnapshot();
});
//# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbIi4uLy4uL3NyYy9zb2xhcml6ZWQtZGFyay9zb2xhcml6ZWQtZGFyay50ZXN0LnRzIl0sIm5hbWVzIjpbInRlc3QiLCJleHBlY3QiLCJzb2xhcml6ZWREYXJrIiwidG9NYXRjaFNuYXBzaG90Il0sIm1hcHBpbmdzIjoiOztBQUFBOzs7O0FBRUFBLElBQUksQ0FBQyxzQ0FBRCxFQUF5QyxZQUFZO0FBQ3JEQyxFQUFBQSxNQUFNLENBQUNDLGlCQUFELENBQU4sQ0FBc0JDLGVBQXRCO0FBQ0gsQ0FGRyxDQUFKIiwic291cmNlc0NvbnRlbnQiOlsiaW1wb3J0IHNvbGFyaXplZERhcmsgZnJvbSAnLi9pbmRleCc7XG5cbnRlc3QoJ1NvbGFyaXplZCBEYXJrIHNob3VsZCBtYXRjaCBzbmFwc2hvdCcsICgpOiB2b2lkID0+IHtcbiAgICBleHBlY3Qoc29sYXJpemVkRGFyaykudG9NYXRjaFNuYXBzaG90KCk7XG59KTtcbiJdfQ==