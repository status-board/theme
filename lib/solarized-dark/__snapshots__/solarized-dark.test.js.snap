// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Solarized Dark should match snapshot 1`] = `
Object {
  "charts": Object {
    "axis": Object {
      "legendColor": "#90a2a4",
      "textColor": "#90a2a4",
      "tickColor": "#90a2a4",
    },
    "colors": Array [
      "#00b2b0",
      "#24dead",
      "#d9d356",
      "#f1c4c2",
      "#dfbbe8",
    ],
    "grid": Object {
      "stroke": "#073642",
    },
    "tooltip": Object {
      "background": "#073c49",
      "borderRadius": 0,
      "boxShadow": "0 1px 3px rgba(0, 0, 0, 0.5)",
      "color": "#bed0d2",
      "fontSize": "1.4vmin",
    },
  },
  "colors": Object {
    "background": "#073642",
    "failure": "#dc322f",
    "icon": "#0badc2",
    "success": "#859900",
    "text": "#90a2a4",
    "textHighlight": "#eee8d5",
    "unknown": "#495b71",
    "warning": "#b58900",
  },
  "dashboard": Object {
    "header": Object {
      "background": "#002b36",
      "title": Object {
        "color": "#eee8d5",
        "fontSize": "1.8vmin",
      },
    },
    "player": Object {
      "slash": Object {
        "color": "#0badc2",
        "margin": "0 0.8vmin",
      },
    },
  },
  "label": Object {
    "addon": Object {
      "background": "#00242f",
      "color": "#93a1a1",
    },
    "background": "#073642",
    "extend": "
        border-radius: 2px;
        ",
    "main": Object {
      "color": "#839496",
    },
  },
  "list": Object {
    "item": Object {
      "hover": Object {
        "background": "#002834",
      },
      "meta": Object {
        "fontSize": "1.4vmin",
      },
    },
  },
  "name": "solarized dark",
  "notifications": Object {
    "item": Object {
      "background": "#073642",
      "color": "#b3c5c7",
      "extend": "
            box-shadow: 0 1px 3px rgba(0, 0, 0, 0.25);
            ",
      "padding": "1.2vmin 2vmin",
    },
  },
  "root": Object {
    "background": "#073642",
    "extend": "
        @import url('https://fonts.googleapis.com/css?family=Space+Mono');
        
        & a {
            text-decoration: underline;
        }
        ",
    "fontFamily": "'Space Mono', Consolas, monospace",
    "fontSize": "1.6vmin",
    "lineHeight": "2.8vmin",
  },
  "typography": Object {
    "default": Object {
      "default": Object {
        "fontFamily": "'Space Mono', Consolas, monospace",
        "fontSize": "1.8vmin",
        "fontWeight": 400,
        "lineHeight": "1.5em",
      },
      "small": Object {
        "fontSize": "1.6vmin",
      },
      "strong": Object {
        "fontWeight": 600,
      },
    },
    "display": Object {
      "default": Object {
        "fontFamily": "'Space Mono', Consolas, monospace",
        "fontSize": "1.8vmin",
        "lineHeight": "1.4em",
      },
    },
    "mono": Object {
      "default": Object {
        "fontFamily": "'Space Mono', Consolas, monospace",
        "fontSize": "1.8vmin",
        "lineHeight": "1.4em",
      },
    },
  },
  "widget": Object {
    "background": "#002b36",
    "header": Object {
      "count": Object {
        "color": "#eee8d5",
      },
      "height": "5vmin",
      "icon": Object {
        "color": "#0badc2",
        "fontSize": "2vmin",
      },
      "subject": Object {},
    },
    "wrapper": Object {
      "padding": "0.6vmin",
    },
  },
}
`;
